plat/rockchip/rk3399/plat_sip_calls.c

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+#include <stdint.h>
```

Comment:
Minor change - includes to be in alphabetical order

Reply:
Done with the latest revision!